.bowerrc

```diff
@@ -0,0 +1,3 @@
+{
+    "directory": "web/assets/vendor"
```

Comment:
I know this is just a POC but I would suggest not putting these inside the web folder. The resulting assets should be compiled into the web folder as part of a build process.